microsoft/testsuites/core/cpu.py

```diff
@@ -0,0 +1,71 @@
+from lisa import testsuite
+from lisa.base_tools.cat import Cat
+from lisa.base_tools.uname import KernelVersion, Uname
+from lisa.node import Node
+from lisa.tools.lscpu import Lscpu
+from lisa.util import SkippedException
+
+NUMA_OFF = "numa=off"
+
+
+@testsuite.TestSuiteMetadata(
+    area="cpu",
+    category="functional",
+    description="""
+    This test suite is used to run CPU related tests.
+    """,
+)
+class CPU(testsuite.TestSuite):
+    @testsuite.TestCaseMetadata(
+        description="""
+        This test case will check that L3 cache is correctly mapped
+        to NUMA node.
+        Steps:
+        1. Check if NUMA is disabled in commandline. If disabled,
+        and kernel version is <= 2.6.37, test is skipped as kernel
+        has no support for NUMA.
```

Comment:
is that possible to add commit (link) which supports NUMA here?

Reply:
This is unsupported b hyperv : https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/best-practices-for-running-linux-on-hyper-v#numa. I can move this link to metadata